src/core/js/views/menuItemView.js

```diff
@@ -0,0 +1,43 @@
+define([
+  'core/js/views/adaptView'
+], function(AdaptView) {
+
+  var MenuItemView = AdaptView.extend({
+
+    attributes: function() {
+      return AdaptView.resultExtend('attributes', {
+        'role': 'listitem',
+        'aria-labelledby': this.model.get('_id') + '-heading'
+      }, this);
+    },
+
+    className: function() {
+      return [
+        'menu-item',
+        this.constructor.className,
+        this.model.get('_id'),
+        this.model.get('_classes'),
+        this.setVisibility(),
+        this.setHidden(),
+        (this.model.get('_isVisited') ? 'is-visited' : ''),
```

Comment:
In router.js

I actually don't think `_isVisited` is very useful since it only works in the current session. We should probably changed to `_isStarted` - but if feel that's a conversation for another day!